spec/tests/yaml/swiftnav/sbp/observation/test_MsgSvAzEl.yaml

```diff
@@ -0,0 +1,227 @@
+description: Unit tests for swiftnav.sbp.observation MsgSvAzEl
```

Comment:
We'll need to run the other generators to generate test code for the other languages